### PR TITLE
fix(rust): fail background node creation if passed identity does not …

### LINKED ISSUE #7625: `ockam node create --identity X` should fail if `X` doesn't exist
### Observed behavior

```
sanjo@mac ~ % ockam node create alice --identity alice
       Creating Node alice...

     ✔ Node alice created successfully

       To see more details on this node, run:
       ockam node show
```

### Steps to reproduce

Run
```
node create alice --identity alice
```
From a clean state

### Desired behavior

Node creation should fail and report a useful error to the user

### Ockam Version

0.116.0

### PR DESCRIPTION
**Pull request by oguseynov**
…exist

Closes #7625 and introduces behaviour that is consistent with one in foreground node creation.

## Current behavior

Background node creation with identity shows success that is not actually true. No error is displayed to user.

## Proposed changes

Having the same behaviour as foreground node creations has: check if identity exists, if not fail with proper error.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

cla accepted in https://github.com/build-trust/ockam/pull/7703

### CODE REVIEW

**Comment by oguseynov** [comment]
It is my first os pr, so I could miss something.

**Comment by oguseynov** [comment]
Rebased against current develop and re-pushed for making lint checks pass.

File: implementations/rust/ockam/ockam_command/tests/bats/nodes.bats [hunk 1]
```diff
@@ -92,6 +92,18 @@ force_kill_node() {
...
+  n="$(random_str)"
+  i=$(random_str)
+  run_failure "$OCKAM" node create "$n" --identity "$i"
+}
+
+@test "node - fail to create foreground node when not existing identity is passed" {
+  n="$(random_str)"
+  i=$(random_str)
+  run_failure "$OCKAM" node create "$n" -f --identity "$i"
+}
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/tests/bats/nodes.bats]: Maybe we could merge both tests into one, and also remove the optional node name argument.

**Review by adrianbenavides — COMMENTED** [review]
Hey @oguseynov, thanks for submitting this PR! It looks good, just a minor detail about the test and will be ready to merge 👍

File: implementations/rust/ockam/ockam_command/tests/bats/nodes.bats (hunk 1 above)
Comment (oguseynov) [reply implementations/rust/ockam/ockam_command/tests/bats/nodes.bats]: sure, done

**Comment by oguseynov** [comment]
Addressed comment regarding tests.

**Review by adrianbenavides — APPROVED** [review]
Awesome, ready to merge 🚀 

Thanks again @oguseynov !

**Comment by oguseynov** [comment]
Thanks for great first oss commit experience 🙏

**Pull request merged**